nss-day-cohort-48/giffygram-hotbot
# R bot

#### Changes Made

1. Modified the Footer.JS module to render the footer in dom.​

#### Steps to Review

1. Checkout this branch locally.
   ```
   git fetch --all
   git checkout branchname to RBot
   ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
   > Make sure footer renders on the dom.
